src/config.rs

```diff
@@ -476,11 +483,13 @@ pub(crate) struct SpotifydConfig {
 }
 
 pub(crate) fn get_internal_config(config: CliConfig) -> SpotifydConfig {
+    let no_audio_cache = !config.shared_config.no_audio_cache;
```

Comment:
Last request :) Would you mind renaming `let no_audio_cache` to `let audio_cache`? Makes more sense since the `no_audio_cache` negated determines if the `audio_cache` is used.

Reply:
If CI passes afterwards I'll merge it :smile:

Reply:
Sure thing! :)